packages/docs-gesture-handler/docs/gestures/_shared/base-gesture-config.md

```diff
@@ -27,42 +56,65 @@ Specifying `width` or `height` is useful if we only want the gesture to activate
 
 **IMPORTANT:** Note that this parameter is primarily designed to reduce the area where gesture can activate. Hence it is only supported for all the values (except `width` and `height`) to be non positive (0 or lower). Although on Android it is supported for the values to also be positive and therefore allow to expand beyond view bounds but not further than the parent view bounds. To achieve this effect on both platforms you can use React Native's View [hitSlop](https://reactnative.dev/docs/view.html#hitslop) property.
 
-### `withRef(ref)`
+### testID
 
-Sets a ref to the gesture object, allowing for interoperability with the old
-API.
-
-### `withTestId(testID)`
+```ts
+testID: string;
+```
 
 Sets a `testID` property for gesture object, allowing for querying for it in tests.
 
-### `cancelsTouchesInView(value)` (**iOS only**)
+### cancelsTouchesInView (**iOS only**)
+
+```ts
+cancelsTouchesInView: boolean | SharedValue<boolean>;
+```
 
 Accepts a boolean value.
 When `true`, the gesture will cancel touches for native UI components (`UIButton`, `UISwitch`, etc) it's attached to when it becomes [`ACTIVE`](/docs/fundamentals/states-events#active).
 Default value is `true`.
 
-### `runOnJS(value: boolean)`
+### runOnJS
+
+```ts
+runOnJS: boolean | SharedValue<boolean>;
+```
 
 When `react-native-reanimated` is installed, the callbacks passed to the gestures are automatically workletized and run on the UI thread when called. This option allows for changing this behavior: when `true`, all the callbacks will be run on the JS thread instead of the UI thread, regardless of whether they are worklets or not.
 Defaults to `false`.
 
-### `simultaneousWithExternalGesture(otherGesture1, otherGesture2, ...)`
+### simultaneousWith
+
+```ts
+simultaneousWith: Gesture | Gesture[]
+```
 
 Adds a gesture that should be recognized simultaneously with this one.
 
 **IMPORTANT:** Note that this method only marks the relation between gestures, without [composing them](/docs/fundamentals/gesture-composition). [`GestureDetector`](/docs/gestures/gesture-detector) will not recognize the `otherGestures` and it needs to be added to another detector in order to be recognized.
 
-### `requireExternalGestureToFail(otherGesture1, otherGesture2, ...)`
+### requireToFail
+
+```ts
+requireToFail: Gesture | Gesture[]
+```
 
 Adds a relation requiring another gesture to fail, before this one can activate.
```

Comment:
Hmm, looks like `requireToFail` is missing the composition warning, which `block` and `simultaneousWith` have.